README.md

```diff
@@ -57,24 +57,55 @@ Oak middleware for JWT
   ```ts
   import { jwtMiddleware, OnSuccessHandler, IgnorePattern } from "https://raw.githubusercontent.com/halvardssm/oak-middleware-jwt/master/mod.ts"
   import { RouterMiddleware } from "https://deno.land/x/oak/mod.ts";
-  
+
   const app = new Application<ApplicationState>();
-  
+
   const ignorePatterns: IgnorePattern[] = ["/baz", /buz/, {path: "/biz", methods: ["GET"]}]
 
   app.use(jwtMiddleware<Middleware>({key: "foo", ignorePatterns}));
-  
+
   await app.listen(appOptions);
   ```
 
 ## Options
 
-* key: string; // See the djwt module for Validation options
-* algorithm: Algorithm | Algorithm[]; // See the djwt module for Validation options
+* secret: string; // See the djwt module for Validation options
```

Comment:
Happy to revert this, of course. It was just for you to get what I was trying to say.

Reply:
I looked at the djwt module, and it seems like they use the terms interchangeably, but that the methods use `key` as the variable name. Am I missing something in regards to why you want to change this? I am fine with changing this, I am just curious to why 😄